CaUWMET/src/demandsLogic/readInputDemands.py

```diff
@@ -0,0 +1,81 @@
+
+
+import os
+import numpy as np
+import pandas as pd
+from totalDemandsLogic import (
+    returnHydroRegion, reclassifyYearType,
+)
+
+# Input directories and filenames
+dirname = os.path.dirname(__file__)
+totalDemandsCSV = "../inputData/totalDemands.csv"
+hydroYearTypeCSV = "../inputData/hydrologyAssumptions.csv"
+baseConservationCSV = "../inputData/baseLongTermConservation.csv"
+inputDemandsFile = os.path.join(dirname, totalDemandsCSV)
+inputHydroYearTypeFile = os.path.join(dirname, hydroYearTypeCSV)
+inputBaseConservationFile = os.path.join(dirname, baseConservationCSV)
+tempHydroRegionDir = r"C:\Users\nudurus\OneDrive - Jacobs\Documents\Projects\2021_05_CVFPP\tasks\20210806_codingDemandsLogic"
```

Comment:
@sainjacobs make this file path dynamic, it's currently hard-coded to your local machine.